app/src/main/java/com/infomaniak/mail/ui/MainViewModel.kt

```diff
@@ -302,14 +300,76 @@ class MainViewModel(application: Application) : AndroidViewModel(application) {
         refreshThreads()
     }
 
+    //region Open Thread
+    fun openThread(threadUid: String) = liveData(Dispatchers.IO) {
```

Comment:
Why did you move it to here?

Reply:
Because I moved `markAsSeen()` from ThreadController to MainViewModel.